installation/sql/postgresql/base.sql

```diff
@@ -1186,6 +1193,12 @@ INSERT INTO "#__workflow_transitions" ("id", "asset_id", "published", "ordering"
 (4, 61, 1, 4, 1, 'Archive', '', -1, 1, '{"publishing":"2"}'),
 (5, 62, 1, 5, 1, 'Feature', '', -1, 1, '{"featuring":"1"}'),
 (6, 63, 1, 6, 1, 'Unfeature', '', -1, 1, '{"featuring":"0"}'),
-(7, 64, 1, 7, 1, 'Publish & Feature', '', -1, 1, '{"publishing":"1","featuring":"1"}');
+(7, 64, 1, 7, 1, 'Publish & Feature', '', -1, 1, '{"publishing":"1","featuring":"1"}'),
+(8, 92, 1, 1, 2, 'Unpublish', '', -1, 2, '{"publishing":"0"}'),
+(9, 91, 1, 2, 2, 'Publish', '', -1, 2, '{"publishing":"1"}'),
+(10, 93, 1, 3, 2, 'Trash', '', -1, 2, '{"publishing":"-2"}');
 
 SELECT setval('#__workflow_transitions_id_seq', 8, false);
```

Comment:
```suggestion
SELECT setval('#__workflow_transitions_id_seq', 11, false);
```
See explanation in some of the above review suggestions.